Bump liburing and remove colliding file

nikitamikhaylov opened the pull request:
Just cosmetics, for more detailed explanation please take a look here: https://github.com/ClickHouse/liburing/pull/1

### Changelog category (leave one):

- Not for changelog (changelog entry is not required)


### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
...

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

<!---
Directly edit documentation source files in the "docs" folder with the same pull-request as code changes

or

Add a user-readable short description of the changes that should be added to docs.clickhouse.com below.

At a minimum, the following information should be added (but add more as needed).
- Motivation: Why is this function, table engine, etc. useful to ClickHouse users?

- Parameters: If the feature being added takes arguments, options or is influenced by settings, please list them below with a brief explanation.

- Example use: A query or command.
-->


> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

#### CI Settings (Only check the boxes if you know what you are doing):
- [ ] <!---ci_set_required--> Allow: All Required Checks
- [ ] <!---ci_include_stateless--> Allow: Stateless tests
- [ ] <!---ci_include_stateful--> Allow: Stateful tests
- [ ] <!---ci_include_integration--> Allow: Integration Tests
- [ ] <!---ci_include_performance--> Allow: Performance tests
- [ ] <!---ci_set_builds--> Allow: All Builds
- [ ] <!---batch_0_1--> Allow: batch 1, 2 for multi-batch jobs
- [ ] <!---batch_2_3--> Allow: batch 3, 4, 5, 6 for multi-batch jobs
---
- [ ] <!---ci_exclude_style--> Exclude: Style check
- [ ] <!---ci_exclude_fast--> Exclude: Fast test
- [ ] <!---ci_exclude_asan--> Exclude: All with ASAN
- [ ] <!---ci_exclude_tsan|msan|ubsan|coverage--> Exclude: All with TSAN, MSAN, UBSAN, Coverage
- [ ] <!---ci_exclude_aarch64|release|debug--> Exclude: All with aarch64, release, debug
---
- [ ] <!---ci_include_fuzzer--> Run only fuzzers related jobs (libFuzzer fuzzers, AST fuzzers, etc.)
- [ ] <!---ci_exclude_ast--> Exclude: AST fuzzers
---
- [ ] <!---do_not_test--> Do not test
- [ ] <!---woolen_wolfdog--> Woolen Wolfdog
- [ ] <!---upload_all--> Upload binaries for special builds
- [ ] <!---no_merge_commit--> Disable merge-commit
- [ ] <!---no_ci_cache--> Disable CI cache

nikitamikhaylov committed:
Remove colliding file from liburing

robot-ch-test-poll1 commented:
<!-- automatic status comment for PR #69166 from ClickHouse/ClickHouse:bump-liburing -->
*This is an automated comment for commit 2110339a20c6eac319db0e51c5971d68901543f8 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/69166/2110339a20c6eac319db0e51c5971d68901543f8/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/1c839ff4b95570cb324cfd9307a0825c32fc50c7/docs_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

nikitamikhaylov commented:
🤦 There is also a problem with sysroot: 
```
~/ClickHouse/ClickHouse/contrib/sysroot                                                                                                                                                                                                                                                            at 18:30:47
❯ git status
HEAD detached at ac37b67
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter/xt_connmark.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter/xt_dscp.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter/xt_mark.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter/xt_rateest.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter/xt_tcpmss.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter_ipv4/ipt_ecn.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter_ipv4/ipt_ttl.h
	modified:   linux-s390x/s390x-linux-gnu/libc/usr/include/linux/netfilter_ipv6/ip6t_hl.h
	modified:   linux-x86_64-musl/include/linux/netfilter/xt_CONNMARK.h
	modified:   linux-x86_64-musl/include/linux/netfilter/xt_DSCP.h
	modified:   linux-x86_64-musl/include/linux/netfilter/xt_MARK.h
	modified:   linux-x86_64-musl/include/linux/netfilter/xt_RATEEST.h
	modified:   linux-x86_64-musl/include/linux/netfilter/xt_TCPMSS.h
	modified:   linux-x86_64-musl/include/linux/netfilter_ipv4/ipt_ECN.h
	modified:   linux-x86_64-musl/include/linux/netfilter_ipv4/ipt_TTL.h
	modified:   linux-x86_64-musl/include/linux/netfilter_ipv6/ip6t_HL.h
```
